experimental/sdg-tracker-un/frontend-tester/UNSD-Website-skeleton/UNSD.Website/ClientApp/src/templates/DataCommons/components/search/Search.tsx

```diff
@@ -336,7 +342,7 @@ const Search = () => {
                 {QUERIES.general.map((q, i) => (
                   <StyledLinkContainer key={i}>
                     <IconSquare color={theme.sdgColors[q.goal - 1]} />
-                    <StyledLink to={`/search?q=${q.query}`}>
+                    <StyledLink to={`search?q=${q.query}`}>
```

Comment:
Just to be sure, should this, and line 362 match line 378?

This code works fine when I click the links locally, but maybe we want to standardized the "to=" field?

Reply:
Good catch! made these consistent